lib/data_update_scripts/20201210163704_set_contact_email_address.rb

```diff
@@ -3,6 +3,8 @@ class SetContactEmailAddress
     def run
       return if SiteConfig.email_addresses[:contact].present?
 
+      # This update does not create and update the contact email,
+      # wrote 20201218080343_update_default_email_addresses as a replacement.
       SiteConfig.email_addresses[:contact] = ApplicationConfig["DEFAULT_EMAIL"]
```

Comment:
This data script which was introduced in https://github.com/forem/forem/pull/11848 did not work - I added the details of how we missed that it wasn't working until deployment in the description. I'm not sure what to do with the data script now that it was run already, do we comment it out?

Reply:
yep, that's what we usually do for duplicate scripts

Reply:
Commented it out and added a reason. ✅